pkgs/development/compilers/ghc/7.10.1.nix

```diff
@@ -13,6 +13,13 @@ let
     ''}
   '';
 
+  # See: https://github.com/haskell/cabal/issues/2320
```

Comment:
The issue https://github.com/haskell/cabal/issues/2320 has been fixed in Cabal-1.22.1.0, and we have 1.22.1.1 in Nixpkgs already, so maybe the whole `fetchgit` bit isn't necessary?